Jenkinsfile

```diff
@@ -76,6 +84,74 @@ pipeline {
                 '''
             }
         }
+        stage ('Package build: 14.04') {
+            environment {
+                BUILD_SERIES = "trusty"
+                ARTIFACT_DIR = "${TMPDIR}${BUILD_SERIES}"
+            }
+            steps {
+                sh '''
+                set -x
+                debuild -S -uc -us
```

Comment:
IIUC here you want to build the source package which is then built with sbuild, and this is why you installed the build-deps in an earlier stage. This is not necessary: the source package can be built by calling `dpkg-source -b .` directly. This is actually what the dpkg-buildpackage(1) manpage suggest ("[...] using dpkg-source directly is always a better [...]"), no build-deps are needed.

But `sbuild` can also be called directly from the deb package source tree, it will call `dpkg-source -b .` by itself. So I think we can skip this step.

Reply:
Excellent thanks @paride. I'll clean this up.